iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -95,14 +98,17 @@
       AlterTableType.ADDCOLS, AlterTableType.REPLACE_COLUMNS, AlterTableType.RENAME_COLUMN,
       AlterTableType.ADDPROPS, AlterTableType.DROPPROPS, AlterTableType.SETPARTITIONSPEC,
       AlterTableType.UPDATE_COLUMNS);
+  private static final List<String> MIGRATION_ALLOWED_SOURCE_FORMATS = Arrays.stream(FileFormat.values())
+      .filter(f -> !f.equals(FileFormat.METADATA))
+      .map(FileFormat::name).map(String::toLowerCase).collect(Collectors.toList());
```

Comment:
I think it would probably be better to list the formats instead i.e. Lists.newArrayList(FileFormat.PARQUET, .. etc). That way it's more readable and safer from a very unlikely scenario a new enum value is added in the future.

Reply:
Good point, will change this to list the 3 supported types only and explicitly